src/components/app/Home.js

```diff
@@ -69,7 +69,10 @@ class UploadButton extends React.PureComponent<UploadButtonProps> {
   render() {
     return (
       <div>
-        <input type="file" ref={this._takeInputRef} onChange={this._upload} />
+        <label className="homeSectionUploadFiles">
+            <input type="file" ref={this._takeInputRef} onChange={this._upload} />
+            Select profile
```

Comment:
The indentation is incorrect for these 2 lines, and this fails our automatic CI (you can see a "red" status on the first page of the pull request). This is our `eslint` hook that runs `prettier` under the hood.

Note that you can configure your editor to automatically run `prettier` or `eslint --fix` (which runs `prettier` under the hood in our project) when saving a file, so that these errors are automatically fixed. Otherwise you can run `yarn lint-fix` in the project which will fix it as well.

Reply:
Please change the text to `Select a profile to open`.